packages/localize/src/number/forceENAUSymbols.js

```diff
@@ -0,0 +1,18 @@
+export function forceENAUSymbols(formattedParts, options) {
+  const result = formattedParts;
+  const numberOfParts = result.length;
+  // Change the symbol from JPY to ¥, due to bug in Chrome
```

Comment:
Can you explain this comment? It's a little confusing to me because it sounds like this is a bug unrelated to what this function solves (issue: unsupported en-AU locale).